prober/file_test.go

```diff
@@ -14,11 +14,15 @@ import (
 	"github.com/ribbybibby/ssl_exporter/config"
 	"github.com/ribbybibby/ssl_exporter/test"
 
+	"github.com/go-kit/log"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 // TestProbeFile tests a file
 func TestProbeFile(t *testing.T) {
+	w := log.NewSyncWriter(os.Stdout)
+	logger := log.NewLogfmtLogger(w)
```

Comment:
I don’t know if there’s a better way to do that rather than repeating this in every test case.

Reply:
You could use `log.NewNopLogger()`:
```
       if err := ProbeFile(log.NewNopLogger(), ctx, glob, module, registry); err != nil {
                t.Fatalf("error: %s", err)
        }
```
Although maybe logs from tests could be valuable?

Otherwise, to curb the repetition you could create a function:
```
func newTestLogger() log.Logger {
	return log.NewLogfmtLogger(log.NewSyncWriter(os.Stdout))
}
```
And then you've got:
```
       if err := ProbeFile(newTestLogger(), ctx, glob, module, registry); err != nil {
                t.Fatalf("error: %s", err)
        }
```

Reply:
I chose the second option :)